docs/change-controlled-docs.rst

```diff
@@ -0,0 +1,233 @@
+.. _writing-ccd:
+
+#######################################
+Writing change-controlled documentation
+#######################################
+
+Change-controlled documents are documents that require approval from a Change Control Board (CCB) before they can be released.
+The project-wide change control process is described in `LPM-19`_, with the process for managing documents being described in `LPM-51`_, and covers those documents using DocuShare handles ``LSE`` or ``LPM`` that must be approved by the Project `CCB`_.
+For Data Management, these documents have the DocuShare handle ``LDM`` and must be approved by the DM CCB using the :ref:`RFC <decision-making-rfc>` process.
+The DM document management process is formally described in `LDM-294`_.
+
+Change-controlled documents can be written using the Word templates defined in `Document-9224`_, but the preference within Data Management is to write change-controlled documents in LaTeX using the ``lsstdoc`` document class provided by the `lsst-texmf`_ package and to develop these documents using Git repositories within the LSST organization on GitHub (https://github.com/lsst).
+This page describes the development and release processes of change-controlled documents produced by DM.
+
+.. _CCB: https://project.lsst.org/groups/ccb/
+.. _Document-9224: https://ls.st/Document-9224
+.. _lsst-texmf: https://lsst-texmf.lsst.io
+.. _LPM-19: https://ls.st/LPM-19
+.. _LPM-51: https://ls.st/LPM-51
+.. _GitHub: https://github.com/lsst
+.. _LDM-294: https://ls.st/LDM-294
+
+.. _ccd-regular:
+
+Regular development
+===================
+
+Develop change-controlled documents using :doc:`DM's standard workflow <../processes/workflow>`.
+That is:
+
+1. Create a ticket branch from ``master``.
+2. Work on the ticket branch.
+3. Have the ticket peer-reviewed.
+4. Rebase and merge the ticket branch to ``master``.
+
+Merging to ``master`` does not denote acceptance by a CCB.
+Instead, it adds to the changeset that will be included in the next CCB review.
+
+.. figure:: ccd-develop.svg
+   :name: fig-ccd-develop
+   :align: center
+   :target: ccd-develop.svg
+   :alt: Git commit tree showing two ticket branches merged to master.
+
+   Example of Git branches during document development.
+   Staff write and edit the document on individual ticket branches that they merge to master, following DM's regular code workflow.
+
+.. _ccd-submit:
+
+Submitting a document to the CCB
+================================
+
+Establishing a new baselined version of the document begins with submitting it to the CCB.
+The release manager is responsible for orchestrating this process on behalf of DM.
```

Comment:
"on behalf of DM" suggests this sentence refers to LSST CCB. Should probably capitalize Release Manager. What does "orchestrating" mean? Which steps below are mediated by the Release Manager? Not everyone who writes an LDM will be able to upload to docushare. They may have to ask someone from DM Systems Engineering to help (or the Release Manager).

Reply:
I'm not capitalizing job titles for consistency with Document-13016 § 2.6 Capitalization of Titles.

Who does what is a good question. I'll pull that conversation back to the main comments rather than this line thread.